schemas/context/experienceevent.schema.json

```diff
@@ -118,6 +118,15 @@
           "title": "Advertising",
           "$ref": "https://ns.adobe.com/xdm/context/advertising",
           "description": "The information related to advertising activity related to the experience event"
+        },
+        "xdm:segmentMemberships": {
```

Comment:
Move xdm:segmentmemberships in experience event to PR request around profile stitch (profile merge)

Reply:
@harleensahni, What would you think about breaking this up into 2 PRs.  One to just add `xdm:segmentMemberships` to ExperienceEvent and a separate one for the payloads?  Then we can more forward with mapping the segment IDs Analytics and others get from AAM now to XDM while the new payload functionality gets worked out.

Reply:
Will do that, see #444 . Will remove segment membership from this one.